cmd/apprepository-controller/main.go

```diff
@@ -78,6 +79,7 @@ func init() {
 	flag.StringVar(&kubeconfig, "kubeconfig", "", "Path to a kubeconfig. Only required if out-of-cluster.")
 	flag.StringVar(&masterURL, "master", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
 	flag.StringVar(&repoSyncImage, "repo-sync-image", "quay.io/helmpack/chart-repo:latest", "container repo/image to use in CronJobs")
+	flag.StringVar(&repoSyncCommand, "repo-sync-cmd", "/chart-repo", "command used to sync/delete repos for repo-sync-image")
```

Comment:
Looks fine. Only confusion here is why it's called a `Command` rather than an `Endpoint`, but small detail. Thanks!

Reply:
it's called command because it's the actual path to the binary to execute (it's not an API endpoint)

Reply:
Oh - I misread. Great. thanks.